description: gcc 6 cannot automatically resolve the this pointer in a lambda
author: Michael Gilbert <mgilbert@debian.org>

--- a/chrome/browser/usb/web_usb_service_impl.cc
+++ b/chrome/browser/usb/web_usb_service_impl.cc
@@ -141,7 +141,7 @@ void WebUsbServiceImpl::OnPermissionRevo
     if (!device_info)
       return true;
 
-    return !HasDevicePermission(*device_info);
+    return !this->HasDevicePermission(*device_info);
   });
 }
 
